polkadot/node/core/dispute-coordinator/src/scraping/tests.rs

```diff
@@ -662,3 +662,251 @@ fn inclusions_per_candidate_properly_adds_and_prunes() {
 		assert!(scraper.get_blocks_including_candidate(&candidate.hash()).len() == 0);
 	});
 }
+
+// ----- Inclusions tests -----
+
+#[test]
+fn inclusions_initialization() {
+	let inclusions = Inclusions::new();
+
+	assert!(inclusions.inclusions_inner.is_empty(), "Expected inclusions_inner to be empty");
+	assert!(
+		inclusions.candidates_by_block_number.is_empty(),
+		"Expected candidates_by_block_number to be empty"
+	);
+}
```

Comment:
nit: I would have skipped this test case.

I believe your point is to have good code coverage and it makes sense in that aspect. However we know that `new` just uses `BTreeMap::new()` and there is no way to end up with a non-empty map. Imho we'd better focus on tests covering actual functionality that might be broken.

This of course is personal point of view/opinion. It's okay to ignore it!

Reply:
I see your point and it definitely wasn't necessary. It's just a habit to make an init test to see if everything indeed intializes the way we are expecting but in that case it's pretty trivial. 

Gonna leave it in as it's already written.

Reply:
Sure! Definitely doesn't hurt to have it.